.github/workflows/tests.yml

```diff
@@ -82,7 +82,9 @@ jobs:
         with:
           timeout_minutes: 5
           max_attempts: 5
-          command: composer update --${{ matrix.stability }} --prefer-dist --no-interaction --no-progress
+          command: |
+            composer require symfony/psr-http-message-bridge:^2.0 nyholm/psr7:^1.0 --no-interaction --no-update
+            composer update --${{ matrix.stability }} --prefer-dist --no-interaction --no-progress
```

Comment:
The optional dependencies are required to run these tests.

Reply:
@timacdonald can we add these to `require-dev` instead please? Also, `nyholm/psr7` needs to be `^1.2` at minimum (see `suggest`).

Reply:
Done!